pkgs/by-name/ta/taxi/package.nix

```diff
@@ -1,58 +1,56 @@
 {
   lib,
   stdenv,
+  desktop-file-utils,
   fetchFromGitHub,
   gobject-introspection,
-  gtk3,
+  gtk4,
+  libadwaita,
   libgee,
   libhandy,
   libsecret,
-  libsoup_2_4,
+  libsoup_3,
   meson,
   ninja,
   nix-update-script,
   pantheon,
   pkg-config,
   python3,
   vala,
-  wrapGAppsHook3,
+  wrapGAppsHook4,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
   pname = "taxi";
-  version = "2.0.2";
+  version = "2.0.2-unstable-2024-12-26";
 
   src = fetchFromGitHub {
     owner = "Alecaddd";
```

Comment:
Do you mind also updating `src` and `meta.homepage` to https://github.com/ellie-commons/taxi/, and `meta.mainProgram` to `io.github.ellie_commons.taxi`?